src/utils/clipboard/getisClipboardAvailable.ts

```
@@ -0,0 +1,3 @@
+export default function getIsClipboardAvailable() {
+  return Boolean(navigator.clipboard && navigator.clipboard.writeText);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)